imod/tests/fixtures/flow_transport_simulation_fixture.py

```diff
@@ -148,7 +177,7 @@ def flow_transport_simulation():
         gwf_model, "species_d", 10.0, 5.0, 0.002
     )
 
-    simulation["flow_solver"] = imod.mf6.Solution(
+    simulation["solver"] = imod.mf6.Solution(
```

Comment:
the string "flow" is hard-coded somewhere, so if  this Solution is not called "solver" bad things happen.

Reply:
The key ``"flow"`` is already used to assign the groundwater model in this example (line 166):

```python
simulation["flow"] = gwf_model
```

However the fact that the key should be "solver" is worrysome. 
Perhaps this is worthy an issue by itself? A quick CTRL+F search shows that the "solver" key is hardcoded in the ``split`` method, which is wrong. We can use the ``_pkg_id`` for that.

Reply:
as discussed, @JoerivanEngelen created an issue for this: #778